src/main/java/ru/practicum/shareit/item/dao/ItemStorageImpl.java

```diff
@@ -0,0 +1,78 @@
+package ru.practicum.shareit.item.dao;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.user.dao.UserStorage;
+import ru.practicum.shareit.user.model.User;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class ItemStorageImpl implements ItemStorage {
+    private final UserStorage userStorage;
+
+    private Map<Integer, Item> items = new HashMap<>();
+    private int counterId = 1;
+
+
+    @Override
+    public List<Item> getItems(Integer ownerId) {
+        User user = userStorage.getUserById(ownerId);
+        return items.values().stream().filter(item -> item.getOwner().equals(user)).collect(Collectors.toList());
```

Comment:
Для лучшей читаемости кода, принято бить стрим по методам:
```suggestion
        return items.values().stream()
            .filter(item -> item.getOwner().equals(user))
            .collect(Collectors.toList());
```